recipes/configs/phi3/mini_full_low_memory.yaml

```diff
@@ -0,0 +1,83 @@
+# Config for single device full finetuning in full_finetune_single_device.py
+# using a Phi3 Mini 4K Instruct
+#
+# This config uses hyperparameters based on small set of experiments and information
+# available on various forums. These are not meant to replicate the numbers
+# from the paper
+#
+# This config assumes that you've run the following command before launching
+# this run:
+#   tune download microsoft/Phi-3-mini-4k-instruct --output-dir ./Phi-3-mini-4k-instruct --hf-token <HF_TOKEN> --ignore-patterns ""
+#
+# The default config uses an optimizer from bitsandbytes. If you do not have it installed,
+# you can install it with
+#   pip install bitsandbytes
+#
+# To launch on a single device, run the following command from root:
+#   tune run full_finetune_single_device --config recipes/config/phi3/mini_full_low_memory.yaml
```

Comment:
```suggestion
#   tune run full_finetune_single_device --config phi3/mini_full_low_memory.yaml
```

Reply:
oh I havent added this to the registry, so this won't work. I'll let you take care of that